plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-color-palette.tsx

```diff
@@ -27,6 +28,11 @@ const SidebarNavigationScreenColorPaletteContent = () => {
 	const hasCreatedOwnColors = !! (
 		user.settings.color && user.settings.color.palette.hasCreatedOwnColors
 	);
+	function handlePanelBodyToggle() {
+		trackEvent(
+			'customize_your_store_assembler_hub_color_palette_create_toggle'
+		);
```

Comment:
Currently, clicking the toggle to open or close it causes the same event. What do you think about adding information on whether it's clicked to open or close it?

Reply:
Works as expected, thanks!